.github/workflows/ci.yml

```
@@ -8,7 +8,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Install build dependencies
-      run: sudo apt-get install -y build-essential autoconf automake libtool cmake lcov
+      run: sudo apt-get install -y build-essential autoconf automake libtool cmake lcov leiningen default-jdk-headless
```

```suggestion
      run: sudo apt-get install -y build-essential autoconf automake libtool cmake lcov
```

So, this is not necessary anymore

---

why isn't it necessary? elle-cli needs leinigen and jdk to build/run

---

oh, i see, because in regular ci, while elle only runs in daily.  part of me thinks the environments should be the same, even if not directly used now, as would make it easier to move tests around without wondering its not running.

---

but jdk is often quite big and it comes with some other packages. I see it is around 250mb++.
Let's not do it. It's too big, it will slow-down the build and contribute to global warming unnecessarily :)